dipy/viz/utils.py

```diff
@@ -159,7 +159,7 @@ def lines_to_vtk_polydata(lines, colors=None):
 
     # Get lines_array in vtk input format
     lines_array = []
-    points_per_line = np.zeros([nb_lines], np.int64)
+    points_per_line = np.zeros([nb_lines], np.intp)
```

Comment:
Yup. Linking back here in the comment.

On Sun, Nov 29, 2015 at 11:50 AM, Matthew Brett <notifications@github.com>
wrote:

> In dipy/viz/utils.py
> <https://github.com/nipy/dipy/pull/789#discussion_r46096969>:
>
> > @@ -159,7 +159,7 @@ def lines_to_vtk_polydata(lines, colors=None):
> >
> >      # Get lines_array in vtk input format
> >      lines_array = []
> > -    points_per_line = np.zeros([nb_lines], np.int64)
> > +    points_per_line = np.zeros([nb_lines], np.intp)
>
> I mean why you are using intp instead of np.int64.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/789/files#r46096969>.
>